annotations/src/main/java/com/statful/client/annotations/Timer.java

```diff
@@ -38,4 +38,9 @@
      * @return An array of aggregations to send to Statful
      */
     Aggregation[] aggregations() default {};
+
+    /**
+     * @return An int of the sampleRate to send to Statful
+     */
+    int sampleRate() default 0;
```

Comment:
I think it'd be better if we set this default as `100`. That way we could avoid the `if` condition below and always send the sample rate as `100` unless specified otherwise - that would effectivelly be the same as not sending it.